### PR TITLE
Add total lines to file tag

### PR DESCRIPTION
**Pull request by masatake**
See the expected.tags.

### CODE REVIEW

File: main/parse.c [hunk 1]
```diff
@@ -1772,6 +1775,26 @@ extern void makeFileTag (const char *const fileName)
 	}
 }
 
+static void makeEOFTag (const char *const fileName)
```
Comment (vhda) [inline main/parse.c]: Why not merge `makeEOFTag` into `makeFileTag` and only call the later where the first is now called?

Comment (masatake) [inline main/parse.c]: I see. I will introduce makeFileComon static function to merge the both code.

Comment (vhda) [inline main/parse.c]: Please check my commit at https://github.com/vhda/ctags/tree/masatake-add-total-lines-to-file-tag

Comment (masatake) [inline main/parse.c]: Oh, I see. Your one is better. Please, merge yours.

Comment (vhda) [inline main/parse.c]: You already have the PR opened! :)
Just drop your "refactor" commit and cherry-pick my extra commit. Everything should be ok after that, or so I hope.

**Pull request merged**